integration_test/version_test.go

```diff
@@ -36,7 +56,7 @@ func TestKrewVersion(t *testing.T) {
 		}
 		optionValue := lineSplit.Split(line, 2)
 		if len(optionValue) < 2 {
-			t.Errorf("Expect `krew version` to output `OPTION VALUE` pair separated by spaces, got: %v", optionValue)
+			return errors.Errorf("Expected `krew version` to output `OPTION VALUE` pair separated by spaces, got: %v", optionValue)
```

Comment:
```suggestion
			return errors.Errorf("`%v` is not an `OPTION VALUE` pair separated by spaces", optionValue)
```
It is much better to start errors with lower case